### PR TITLE
Add an adapter for benchmark.oss_ci_benchmark_v3

### PR DESCRIPTION
**Pull request by huydhn**
The schema comes from https://github.com/pytorch/test-infra/blob/main/torchci/clickhouse_queries/oss_ci_benchmark_v3/query.sql.  An example S3 path is `s3://ossci-benchmarks/v3/pytorch/pytorch/11850871071/33027181871/add_loop_eager_dynamic.json`

I think we should figure out how to test changes to these replicator lambdas.  Otherwise, we might lose some data if they break.  Any thoughts?

### CODE REVIEW

**Comment by huydhn** [comment]
@clee2000 Do you know if the file on S3 needs to be public accessible for the replication to work? i.e. https://ossci-benchmarks.s3.us-east-1.amazonaws.com/v3/pytorch/pytorch/11831781500/33022921916/add_loop_eager_dynamic.json needs to be public

**Comment by clee2000** [comment]
> @clee2000 Do you know if the file on S3 needs to be public accessible for the replication to work? i.e. https://ossci-benchmarks.s3.us-east-1.amazonaws.com/v3/pytorch/pytorch/11831781500/33022921916/add_loop_eager_dynamic.json needs to be public

It needs either public access or for the clickhouse role to have access. You can test if it works by performing the query in the query console or by running the script locally

**Comment by huydhn** [comment]
Yeah, I have been able to run this locally and add some records to that table from S3

File: aws/lambda/clickhouse-replicator-s3/lambda_function.py [hunk 1]
```diff
@@ -379,6 +379,72 @@ def torchao_perf_stats_adapter(table, bucket, key) -> None:
...
+        )
+    ),
+    `metric` Tuple(
+        name String,
+        benchmark_values Array(Float32),
+        target_value Float32,
+        extra_info Map(String, String)
+    ),
+    """
+    general_adapter(table, bucket, key, schema, ["none"], "JSONEachRow")
```
Comment (clee2000) [inline aws/lambda/clickhouse-replicator-s3/lambda_function.py:445]: don't you use gzip in the upload?

**Review by clee2000 — APPROVED** [review]

File: aws/lambda/clickhouse-replicator-s3/lambda_function.py (hunk 1 above)
Comment (huydhn) [reply aws/lambda/clickhouse-replicator-s3/lambda_function.py:445]: Ok, I thought that this is the format of the source files on S3, they aren't gzipped, but let me add gzip here

**Pull request merged**